tests/rules/javascript/test.no_implied_eval.js

```diff
@@ -0,0 +1,157 @@
+import { VALIDATION_WARNING } from 'const';
+import { singleLineString } from 'utils';
+import JavaScriptScanner from 'scanners/javascript';
+import { NO_IMPLIED_EVAL } from 'messages';
+
+
+// These rules were mostly copied and adapted from eslint.
+// Please make sure to keep them up-to-date and report upstream errors.
+describe('no_implied_eval', () => {
+  const expectedErrorMessage = singleLineString`
+    Implied eval. Consider passing a function instead of a string.`;
+
+  const validCodes = [
+    // normal usage
+    'setInterval(function() { x = 1; }, 100);',
```

Comment:
We had a test in the old code for an arrow function it would be nice to check that still works.